commit 215f04ebb34488b1c22ee6e78c6917aa1ddb0261
Author: Fabio Erculiani <lxnay@sabayon.org>
Date:   Mon Oct 24 16:20:35 2011 +0200

    [entropy.core] fix variables unpack while scanning self.__setting_dirs

diff --git a/lib/entropy/core/settings/base.py b/lib/entropy/core/settings/base.py
index 87faeb9..a8053a6 100644
--- a/lib/entropy/core/settings/base.py
+++ b/lib/entropy/core/settings/base.py
@@ -226,7 +226,8 @@ class SystemSettings(Singleton, EntropyPluginStore):
             self.__setting_files['system_mask'],
             self.__setting_files['splitdebug'],
         ])
-        for setting_id, dir_sett, auto_update in self.__setting_dirs.items():
+        for setting_id, setting_data in self.__setting_dirs.items():
+            conf_dir, dir_sett, auto_update = setting_data
             if not auto_update:
                 continue
             for conf_file, mtime_conf_file in dir_sett:
